Index: src/main/java/org/jmc/pintfinder/Homepage.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/jmc/pintfinder/Homepage.fxml b/src/main/java/org/jmc/pintfinder/Homepage.fxml
new file mode 100644
--- /dev/null	(date 1743439380663)
+++ b/src/main/java/org/jmc/pintfinder/Homepage.fxml	(date 1743439380663)
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import java.util.*?>
+<?import javafx.scene.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<AnchorPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:controller="org.jmc.pintfinder.Homepage"
+            prefHeight="400.0" prefWidth="600.0">
+
+</AnchorPane>
